google/cloud/alloydb/connector/instance.py

```diff
@@ -56,21 +78,14 @@ def __init__(
         keys: asyncio.Future[Tuple[rsa.RSAPrivateKey, str]],
     ) -> None:
         # validate and parse instance_uri
-        instance_uri_split = instance_uri.split("/")
-        # should take form "projects/<PROJECT>/locations/<REGION>/clusters/<CLUSTER>/instances/<INSTANCE>"
-        if len(instance_uri_split) == 8:
-            self._instance_uri = instance_uri
-            self._project = instance_uri_split[1]
-            self._region = instance_uri_split[3]
-            self._cluster = instance_uri_split[5]
-            self._name = instance_uri_split[7]
-        else:
-            raise ValueError(
-                "Arg `instance_uri` must have "
-                "format: projects/<PROJECT>/locations/<REGION>/clusters/<CLUSTER>/instances/<INSTANCE>, "
-                f"got {instance_uri}."
-            )
-
+        (
+            self._project,
+            self._region,
+            self._cluster,
+            self._name,
+        ) = _parse_instance_uri(instance_uri)
+
```

Comment:
is the linter adding the `()` around this tuple? I don't think we need them as the function can just unpack the tuple into the appropriate variables.
```suggestion
            self._project, self._region, self._cluster, self._name = _parse_instance_uri(instance_uri)
```

Reply:
yes @jackwotherspoon , the tuple was causing the formatting. I have removed it in the latest push.